AxonDeepSeg/download_tests.py

```diff
@@ -10,22 +10,27 @@ def download_tests(destination = None):
         destination = convert_path(destination)
         test_files_destination = destination / "__test_files__"
 
-    url_tests = "https://osf.io/vqt94/?action=download"  # URL of TEM model hosted on OSF storage with the appropriate versioning on OSF
+
+    # retrieve latest release identifier from github API
+    response_tests_files = requests.get("https://api.github.com/repos/axondeepseg/data-testing/releases/latest")
+    folder_name_test_files = Path("data-testing-"+str(response_tests_files.json()['tag_name']))
+
+    url_tests = "https://github.com/axondeepseg/data-testing/archive/refs/tags/"+str(response_tests_files.json()['tag_name'])+".zip"   # URL of TEM model hosted on OSF storage with the appropriate versioning on OSF
```

Comment:
If I am not mistaken, for the test files, as for #533, we will need the tag_name hardcoded in the URL.
If this is indeed the case I suggest you use the same trick I used (comparing files in the current directory before/after the download).

Reply:
> If I am not mistaken, for the test files, as for #533, we will need the tag_name hardcoded in the URL.
If this is indeed the case I suggest you use the same trick I used (comparing files in the current directory before/after the download).

Yes, you're right. It needs to be updated. I'll update this. 

> edit: also if we indeed want a static tag_name in the URL, we should wait for @Stoyan-I-A to modify the files in the gh repo before merging this PR

Yes you are absolutely right, we should wait for @Stoyan-I-A or me to update the data-testing repo.